config/jobs/kubernetes/kops/kops-presubmits-network-plugins.yaml

```diff
@@ -0,0 +1,314 @@
+presubmits:
+  kubernetes/kops:
+
+  - name: pull-kops-e2e-cni-amazonvpc
+    branches:
+    - master
+    run_if_changed: '^upup\/models\/cloudup\/resources\/addons\/networking\.amazon-vpc-routed-eni\/'
+    skip_report: false
```

Comment:
can we make these `optional: true` to start? that way we dont bombard all the open PRs with test failures if theres an issue with their setup.

Reply:
it will affect only those with the path and there are none at the moment

Reply:
ah ok, that sounds good